.github/workflows/ci.yml

```diff
@@ -282,10 +282,15 @@ jobs:
       matrix:
         target:
           - runner-type: gpu
-            iree-ctest-label-regex: ^requires-gpu|^driver=vulkan$|^driver=cuda$
+            iree-ctest-label-regex:
+              --label-regex ^requires-gpu|^driver=vulkan$|^driver=cuda$
+            iree-nvidia-sm80-tests-disable: 1
+          - runner-type: 2-gpus
```

Comment:
We haven't had a solid plan to add 2 GPU VM (I assume it will be a2-highgpu-2g). Although there are some requests from benchmarking.

 For the cost, I'd like to only have 1 VM in postsubmit at the beginning (its cost = 2x 1GPU VM). If there is a reasonable reason to run 2 GPUs tests in CI, I'm fine to add it). From the PR description, I'm not quite clear about why these tests need to run on 2GPUs, it will be great to give out more details to help us decide (like what do these tests cover in addition to single GPU tests).

Reply:
In NCCL you can't put more than one rank per GPU. I will add explanation in the test.

Reply:
To make progress on this PR, I'd suggest adding the test without a CI job that runs it. We can add a postsubmit CI job with multiple GPUs separately, but that work can happen independently.

(Seeing the proposed CI changes does help with the review, at least, but I think some of them can be reverted here / moved to a separate PR)

Reply:
I removed the 2-GPU test from the CI job and the installation of OpenMPI from the Docker files.